modules/react/action-bar/lib/ActionBarList.tsx

```diff
@@ -8,14 +8,13 @@ import {
   styled,
   StyledType,
 } from '@workday/canvas-kit-react/common';
-import {HStack} from '@workday/canvas-kit-react/layout';
+import {Flex} from '@workday/canvas-kit-react/layout';
 import {useOverflowListMeasure, useListRenderItems} from '@workday/canvas-kit-react/collection';
 
 import {useActionBarModel} from './useActionBarModel';
 
-// Use `Partial` here to make `spacing` optional
 export interface ActionBarListProps<T = any>
-  extends Omit<Partial<ExtractProps<typeof HStack, never>>, 'children'> {
+  extends Omit<ExtractProps<typeof Flex, never>, 'children'> {
```

Comment:
I don't think `Flex` requires `children`, i think you can simplify this type to be `export interface ActionBarListProps<T = any> extends Flex {`

Reply:
sorry,  extract props